chart/compass/templates/subaccount-fetcher-job.yaml

```diff
@@ -0,0 +1,272 @@
+{{ $outerScope := . }}
```

Comment:
This is basically the same chart as the old TF job, except for some missing properties, can we somehow reuse the existing one?